_layouts/index.html

```diff
@@ -0,0 +1,94 @@
+---
+layout: base
+---
+
+
+<!-- style="background-color: blue;" -->
+  <div class="container content {{ layout.css }} {{ page.css }}">
+        <!-- section1 START-->
+        <div class="columns section1 has-text-centered-mobile is-vcentered is-flex" >
+            <div class="column section1_text">
+                <p class="hallo_text">Hallo!</p>
+                <h1 class="is-size-3-mobile welcome_text">Welcome to our online & interactive game
+                    on DNA data analysis</h1>
+                <p class="come_join_text">Learn Bioinformatics online through interesting games! Come join us now!</p>
+                <a href="{{ site.baseurl }}/game/intro/" class="button is-large is-fullwidth">Get Started</a>
+            </div>
+            <div class="column section1_img is-hidden-mobile">
+                <img src="{{ site.baseurl }}/images/hero_img.png" alt="">
+            </div>
+        </div>
+        <!-- section1 END-->
+
+        <!-- section2 START-->
+        <div class="section2 has-text-centered-mobile">
+            <div class="section2_text">
+                <p>Games</p>
+                <p class="is-size-3-mobile">Three exciting levels help you to learn!</p>
+                <p>Rocket your bioinformatics knowledge. Learn, play, enjoy, All at the same time.</p>
+            </div>
+            <ul class="columns section2_cards is-multiline">
+                <!-- card 1 -->
+                <li class="column">
+                    <div class="card  is-fullheight">
+                        <div class="card-image">
+                            <figure class="image is-4by3">
+                                <img src="{{ site.baseurl }}/images/alien.png" alt="Sam, the alien">
+                            </figure>
+                        </div>
+                        <div class="card-content">
+                            <p class="title is-4">Sam, the alien.</p>
+                            <div class="content">
+                                Meet Sam and help him to understand life on eath. Together you will find out what every organism on earth has in common and other fascinating biological facts you need to know before performing data analysis.
+                            </div>
+                        </div>
+                        <footer class="card-footer">
+                            <a href="#" class="button is-info is-medium is-fullwidth">Learn more</a>
```

Comment:
link directly to the games introduction page and not the start button on the same page.